packages/shortest/src/core/runner/index.ts

```diff
@@ -61,29 +60,6 @@ export class TestRunner {
     this.browserManager = new BrowserManager(this.config);
   }
 
-  private async findTestFiles(pattern?: string): Promise<string[]> {
```

Comment:
Once the default value assignment and duplicated logic are removed, there isn't much value having this function.